tests/publiccloud/instance_overview.pm

```diff
@@ -55,5 +60,9 @@ sub run {
     zypper_call("lr");
```

Comment:
we both overlook this line :) 
so please remove one which you have added and just add `-d` here

Reply:
Ohhh good one! I really missed that one at the end :+1: